.Lib9c.Tests/TableData/Pet/PetCostSheetTest.cs

```diff
@@ -1,58 +1,57 @@
 ﻿namespace Lib9c.Tests.TableData.Pet
 {
-    using System;
-    using System.Linq;
-    using Lib9c.Tests.Action;
     using Nekoyume.TableData.Pet;
     using Xunit;
 
     public class PetCostSheetTest
     {
-        private PetCostSheet _petCostSheet;
-
-        public PetCostSheetTest()
-        {
-            if (!TableSheetsImporter.TryGetCsv(nameof(PetCostSheet), out var csv))
-            {
-                throw new Exception($"Not found sheet: {nameof(PetCostSheet)}");
-            }
-
-            _petCostSheet = new PetCostSheet();
-            _petCostSheet.Set(csv);
-        }
-
         [Fact]
-        public void SetToSheet()
+        public void Set()
         {
-            const string content =
-                @"ID,_PET NAME,PetLevel,SoulStoneQuantity,NcgQuantity
+            const string content = @"pet_id,_pet_name,pet_level,soul_stone_quantity,ncg_quantity
 1,D:CC 블랙캣,1,10,0
-        ";
+1,D:CC 블랙캣,2,0,10";
```

Comment:
Can we use actual CSV file, not hard coded sample data?
It would be more helpful if CSV contents are changed.

Reply:
I guess the actual CSV file you said is that contained in `Lib9c/TableCSV/`.
These CSV files already tested when the `Lib9c.Tests.TableSheets` instantiated.

Currently all `XxxSheet.Set()` test just check couple of success cases. And I also think if there are failure cases it would be better.